config/jobs/image-pushing/k8s-staging-cluster-api.yaml

```diff
@@ -10,6 +10,8 @@ postsubmits:
       branches:
         - ^master$
         - ^release-0.2$
+        # this is a regex for semver, from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+        - ^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
```

Comment:
If this is meant to trigger on a tag please keep in mind that treating tags as branches is an unintended and unsupported feature that we may remove or change.

Reply:
Is there an alternative coming up to trigger a build when a tag is pushed?

Reply:
We have discussed this several times — it is a required feature and we will support it as-is until a better solution comes around (at which point we should fix these configs — we know that any file in this directory is potential use case, so they’re easy enough to find here)

Reply:
Thanks for the clarification @Katharine !